src/ralph/deployment/tests/test_deployment.py

```diff
@@ -105,3 +109,20 @@ def _prepare_rack(self):
         super()._prepare_rack()
         self.instance.parent.rack = self.rack
         self.instance.parent.save()
+
+
+@ddt
+class AutocompleteFunctionsTestCase(TestCase):
+    @unpack
+    @data(
+        ([],),
+        ([DataCenterAssetFactory(), DataCenterAssetFactory()],)
```

Comment:
move it to the body of the function (or make it somehow lazy) - now these objects are created when this module is importing and not every table is created then.